src/Command/Command/Market/BookKeeper.php

```diff
@@ -127,21 +144,22 @@ protected function logException(\Exception $e, $logNewTrace = true) : void
      *
      * @return bool
      */
-    protected function isMaintenance() : bool
+    protected function isMaintenance(): bool
     {
-        return false;
-        $ch = \curl_init(\sprintf('https://%s', (new Config())->gemini->api->host));
+        if (!self::$maintenance) { // mostly for IDE to not yell
+            return false;
+        }
```

Comment:
Huh?

Reply:
Nvm I follow now. 

This entire function is old code from early on. I'd rather us rip it out entirely from this class for the time being. Removing any references to it.

Reply:
Sounds good to me